lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -293,6 +293,12 @@ sub scenario {
     return $scenario;
 }
 
+sub scenario_description {
+    my ($self) = @_;
+    my $description = $self->result_source->schema->resultset('TestSuites')->find({ name => $self->TEST})->description;
```

Comment:
DBIx has better means to do that query. Rather than explaining I can push a further commit to your branch.

Reply:
Sure, :) I finally found some solution after looking at the tables, but if you know something better, go ahead.